models/prices/polygon/prices_polygon_tokens.sql

```diff
@@ -68,5 +68,19 @@ FROM
     ('quick-quickswap','polygon','QUICK',0x831753dd7087cac61ab5644b308642cc1c33dc13,18),
     ('dimo-dimo','polygon','DIMO',0xe261d618a959afffd53168cd07d12e37b26761db,18),
     ('gddy-giddy','polygon','GDDY',0x67eb41a14c0fe5cd701fc9d5a3d6597a72f641a6,18),
-    ('val-valeria','polygon','VAL',0x456f931298065b1852647de005dd27227146d8b9,18)
+    ('val-valeria','polygon','VAL',0x456f931298065b1852647de005dd27227146d8b9,18),
+    ('truehn-true-human-nature','polygon','TRUEHN',0x9d9f8a6a6ad70d5670b7b5ca2042c7e106e2fb78,9),
+    ('stg-stargatetoken','polygon','STG',0x2f6f07cdcf3588944bf4c42ac74ff24bf56e7590,18),
+    ('titan-iron-titanium-token','polygon','IRON ',0xaaa5b9e6c589642f98a1cda99b9d024b8407285a,18),
+    ('grt-the-graph','polygon','GRT',0x5fe2b58c013d7601147dcdd68c143a77499f5531,18),
+    ('snx-synthetix-network-token','polygon','SNX',0x50b728d8d964fd00c2d0aad81718b71311fef68a,18),
+    ('sff-sunflower-farm','polygon','SFF',0xdf9b4b57865b403e08c85568442f95c26b7896b0,18),
+    ('dino-dinoswap','polygon','DINO',0xaa9654becca45b5bdfa5ac646c939c62b527d394,18),
+    ('xend-xend-finance','polygon','XEND',0xa649325aa7c5093d12d6f98eb4378deae68ce23f,18),
```

Comment:
Hi @RantumBits, Please verify this token address:
https://polygonscan.com/token/0xa649325aa7c5093d12d6f98eb4378deae68ce23f

Reply:
this should be 0x86775d0b80b3df266af5377db34ba8f318d715ec

I mistakenly grabbed the address from HECO chain from CoinPaprika